Leanplum-SDK/Classes/Managers/Networking/LPRequestSender.m

```diff
@@ -119,36 +120,45 @@ - (void)sendNow:(id<LPRequesting>)request sync:(BOOL)sync
             return;
         }
 
-        [self sendEventually:request];
+        [self sendEventually:request async:!sync];
```

Comment:
Whats going on here with !

Reply:
So the `LPRequestSender:sendNow` actually has `sync` bool, which is used to determine whether the request should be executed sync(true) or async(false), while the `LeanplumRequest` uses similar `async` bool in `sendNow` method which is opposite. I'll have to refactor and align the naming across two classes.